frontend/src/components/PlanForm.css

```diff
@@ -0,0 +1,15 @@
+.form {
+  width: 500px;
+  margin: 40px auto;
+  display: flex;
+  flex-direction: column;
+  gap: 2em;
+}
+
+.error-message {
+  flex: 0 0 100%;
+  flex-wrap: wrap;
+  color: red;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
generally good practice to leave a newline at end of files, which is what this red icon denotes